src/App.js

```diff
@@ -1,24 +1,37 @@
 import logo from './logo.svg';
 import './App.css';
+import styled from 'styled-components';
+
+const ButtonGroup = styled.div`
+  display: flex;
+`
```

Comment:
Unnecessary package, the same job can probably be accomplished with a div or some other css.

Reply:
True.